src/SprykerEco/Yves/Payone/Form/CashOnDeliverySubForm.php

```diff
@@ -38,7 +41,7 @@ public function getPropertyPath(): string
      */
     public function getTemplatePath(): string
     {
-        return PayoneConstants::PROVIDER_NAME . '/' . self::PAYMENT_METHOD;
+        return PayoneConstants::PROVIDER_NAME . '/' . static::PAYMENT_METHOD;
```

Comment:
```suggestion
        return sprintf(static::TEMPLATE_PATH_PATTERN, $this->getProviderName(), $this->getPaymentMethod());
```

Reply:
Seems getTemplacePath could be moved to AbstractPayoneSubForm because it's used in every SubForm.

Reply:
static::TEMPLATE_PATH_PATTERN not found.

Reply:
As i got told in another PR it seems we are not allowed to implement pattern as constant for sprintf. Please ignore this requested change ...